packages/ui/components/calendar/components/event/Empty.tsx

```diff
@@ -0,0 +1,43 @@
+import shallow from "zustand/shallow";
+
+import { useCalendarStore } from "../../state/store";
+import { gridCellToDateTime, GridCellToDateProps } from "../../utils";
+
+export function EmptyCell(props: GridCellToDateProps) {
+  const { onEmptyCellClick, hoverEventDuration } = useCalendarStore(
+    (state) => ({
+      onEmptyCellClick: state.onEmptyCellClick,
+      hoverEventDuration: state.hoverEventDuration,
+    }),
+    shallow
+  );
+
+  const cellToDate = gridCellToDateTime({
+    day: props.day,
+    gridCellIdx: props.gridCellIdx,
+    totalGridCells: props.totalGridCells,
+    selectionLength: props.selectionLength,
+    startHour: props.startHour,
+  });
+
+  return (
+    <div
+      className="group w-full"
+      style={{ height: "1.75rem", overflow: "visible" }}
+      onClick={() => onEmptyCellClick && onEmptyCellClick(cellToDate.toDate())}>
+      {hoverEventDuration !== 0 && (
```

Comment:
Perhaps return null early instead of rendering an empty div that in theory is still clickable?

Reply:
We still need this div to render the grid and also handle the onClick since we moved away from template cells

Reply:
Makes sense! I'll resolve this one.